### ISSUE TITLE
Why React is Listed as Dependency by Default in new Single SPA app Generated w/CLI

### ISSUE THREAD

**Issue by edarioq** [single-spa/create-single-spa#171]
**Describe the bug or question**
A clear and concise description of what the bug or question is.

I see this listed as dependencies in `src` `package.json`:

```
"dependencies": {
    "single-spa": "^5.3.1",
    "@types/jest": "^25.2.1",
    "@types/react": "^16.9.32",
    "@types/react-dom": "^16.9.6",
    "@types/systemjs": "^6.1.0",
    "@types/webpack-env": "^1.15.1"
  }
```

Why is React and React DOM listed as dependencies by default when creating a new app with Single SPA CLI ?

**Comment by joeldenning** [comment]
Hi @edarioq, what options did you choose when you created the app with the CLI? It is expected that react and react-dom would be dependencies for react applications, but they should not be dependencies for non-react microfrontends.

**Comment by edarioq** [comment]
Hi Joel, I used Single SPA CLI to generate the root app. With options like these:

![Screen Shot 2020-08-27 at 8 34 14 AM](https://user-images.githubusercontent.com/1882457/91448944-25537980-e840-11ea-9848-e3242fa5b1b8.png)

After looking at the `package.json` react is listed once again, also Jest.

```
  "dependencies": {
    "single-spa": "^5.3.1",
    "@types/jest": "^25.2.1",
    "@types/react": "^16.9.32",
    "@types/react-dom": "^16.9.6",
    "@types/systemjs": "^6.1.0",
    "@types/webpack-env": "^1.15.1",
    "single-spa-layout": "1.0.0-beta.6"
  }
```

**Comment by joeldenning** [comment]
This looks like a bug. Let's fix it

**Comment by joeldenning** [comment]
Fixed in #172 and released in https://github.com/single-spa/create-single-spa/releases/tag/v1.12.4
